tools/cldr-apps/src/main/java/org/unicode/cldr/web/SurveyForum.java

```diff
@@ -1098,25 +1098,36 @@ private int savePostToDb(PostInfo postInfo) throws SurveyException {
     /**
      * Close all posts in the threads with the given root ids
      *
+     * This is a long-running process (several minutes) if the number of posts is large. There is no
+     * progress feedback on the front end, unfortunately. As a work-around, print progress to Java log.
+     * Only Admin can use this feature. Admin should be able to watch log even on production server.
+     *
+     * This feature could probably be made a hundred times faster by using an sql stored procedure!
```

Comment:
I don't know the whole context, but file a new perf bug for this speed up if needed. 

Also, it could be made async but let's wait until there's a better api substrate. 

It could be just moved to a surveytask if there's no interactive response needed. That would keep it from timing out.

Reply:
It's only run likely to be run once per version, so not a high priority for performance optimization. And yes, the only feedback really required is confirmation that it completed successfully -- and that does now occur, after five or ten minutes :-)